### ISSUE TITLE
Want to contribute? Welcome! Please start here

### ISSUE THREAD

**Issue by sidneymbell** [nextstrain/ncov#267]
# Contributing to translations of our situation reports
## Please see our [Translation Guide](https://github.com/nextstrain/ncov/blob/master/TRANSLATION_DOCS.md) to get started!

# Contributing to software or documentation
## Please see our [Contributor Guide](https://github.com/nextstrain/.github/blob/master/CONTRIBUTING.md) to get started!

**Comment by Maijin** [comment]
Issues aren't really a good for broadcasting message - after a week this one won't be visible anymore.

I recommend you use the standard like CONTRIBUTING.md it will show in the bottom when creating issues as follow:
<img width="547" alt="image" src="https://user-images.githubusercontent.com/1408600/77466528-b2fee900-6e0a-11ea-9330-40144e477eb1.png">

You can also create some issue template which will result in:

**Comment by sidneymbell** [comment]
Yup! We've already got all those implemented, as well. A pinned issue just ensures that people see these links even if they bypass the README, etc.
